liblangutil/SourceLocation.h

```diff
@@ -53,9 +49,9 @@ struct SourceLocation
 
 	bool isEmpty() const { return start == -1 && end == -1; }
 
-	int start;
-	int end;
-	std::shared_ptr<CharStream> source;
+	int start {-1};
```

Comment:
```suggestion
	int start{-1};
```
Or could we perhaps just use `= -1` for some here? It's so much more readable.
What about `int start = {-1}`?